templates/download/iot/intel-iotg.html

```diff
@@ -0,0 +1,71 @@
+{% block meta_copydoc %}https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Download Ubuntu for Intel IoT platforms</h1>
+      <p class="p-heading--4">Ubuntu - now optimized for Intel's IOTG platforms</p>
+      <p>Ubuntu Core 20 and Ubuntu Desktop 20.04 based images for Elkhart Lake and Tiger Lake platforms will be released on October 14th and are currently available as a private alpha.</p>
+      <a class="p-button--positive" href="mailto:sales-iotg@canonical.com&subject=Ubuntu+Core+20+private+alpha+for+Elkhart+Lake+and+Tiger+Lake">Contact us if you'd like early access</a>
+    </div>
+    <div class="col-5 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ab843223-Intel.svg",
+        alt="",
+        width="200",
+        height="132",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Ubuntu on Intel IoT platforms</h2>
+    <p>Intel's portfolio of edge-ready compute and connectivity technologies make it easy to deploy edge applications quickly. Enhanced for IoT, they enable processing at the edge to get critical insights and business value from your data with compute resources where you need them most.</p>
+    <p><strong>Supported processors:</strong></p>
+    <ul class="p-list">
+      <li class="p-list__item is-ticked">
+        Intel Atom&reg; X6000E Series and Intel® Pentium&reg; and Celeron&reg; N and J Series Processors
+      </li>
+      <li class="p-list__item is-ticked">
+        Intel 11th Gen Intel Core&trade; processors
+      </li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h3>Learn more about Intel Atom&reg; X6000E Series and Intel&reg; Pentium&reg; and Celeron&reg; N and J Series Processors</h3>
+      <p>To support the next generation of IoT edge devices, Intel has developed a new line of processors enhanced for IoT—The Intel Atom x6000E Series, Intel Pentium, and Intel Celeron N and J Series processors (formerly known as Elkhart Lake - EHL). These processors build on new levels of CPU, and graphics performances with integrated IoT features, real-time performance, manageability, security, and functional safety.</p>
+      <p><a class="p-link--external" href="https://www.intel.com/content/www/us/en/products/docs/processors/embedded/enhanced-for-iot-platform-brief.html">Read the platform brief</a></p>
```

Comment:
Can you remove the `p` here to remove the margin bottom please? Matching it with the row below.